data/changelog.txt

```diff
@@ -309,17 +335,18 @@ Enable compiling an Android build under Windows.
 Tools: add a Python script to reformat vehicles to use parts arrays.
 Suppress Homebrew update for faster OSX builds.
 Adds problem matchers for json errors, compiler errors, DebugLog emissions, and Catch2 errors.
-Add check enforcing almost-never-auto convention.
+Adds check enforcing almost-never-auto convention.
 Adds a GitHub workflow running flake8 to check the Python scripts.
 
 ## I18N and A11Y:
 Fix translation of martial art initiate message and computer translation.
```

Comment:
```suggestion
Fixes translations of martial art initiate messages and computer translation.
```
"and computer translation" means what? Had been computer translated previously, or what?